NAMESPACE

```diff
@@ -18,9 +18,19 @@ export(package_exports_checked)
 export(postconditions)
 export(preconditions)
 export(present)
+export(print_args)
 export(quickcheck)
 export(should_be_checked)
 export(validate)
 export(validate_)
 import(memoise)
 import(methods)
+importFrom(devtools,as.package)
```

Comment:
If we add this (which is fine), we need to move `devtools` from the `Suggests` to the `Imports` in the `DESCRIPTION` file.